python/lsst/ts/externalscripts/maintel/tma/random_walk.py

```diff
@@ -201,15 +232,17 @@ def set_metadata(self, metadata):
         metadata.duration = self.config.total_time
 
     async def run(self):
-        async for i, az, el in self.random_walk_azel_by_time():
-            await self.checkpoint(f"[{i}] Tracking {az=}/{el=}.")
-            await self.slew_and_track(az, el, target_name=f"random_walk_{i}")
+        async for data in self.random_walk_azel_by_time():
```

Comment:
What does the "by_time" in the method name mean? This seems to pick a random az/el, but I don't see use of time in that method. Perhaps consider a name such as `get_random_walk_azel_target`?

Reply:
The "by_time" means this function will generate a random target for a finite time. I will update the docstring to try to be more explicit. Initially, the time was an input parameter, making it easier to see, but Tiago asked to use the configuration directly inside the method.